pkg/image/imagetag.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+// Tag contains an image and a new tag, which will replace the original tag.
+// Tag usage is deprecated, instead use ImageTag.
+type Tag struct {
```

Comment:
Can we keep the name `ImageTag`?

Reply:
sure, I just wanted to make happy the linter